DeviceManager/DeviceHandler.py

```diff
@@ -134,9 +135,30 @@ def get_devices(req):
         init_tenant_context(req, db)
 
         page_number, per_page = get_pagination(req)
-        page = Device.query.paginate(page=page_number,
-                                     per_page=per_page,
-                                     error_out=False)
+        pagination = {'page': page_number, 'per_page': per_page, 'error_out': False}
+
+        parsed_query = []
+        query = req.args.getlist('attr')
+        for attr in query:
+            parsed = re.search('^(.+){1}=(.+){1}$', attr)
+            parsed_query.append("attrs.label = '{}'".format(parsed.group(1)))
```

Comment:
Just for the record: I found that python 3 supports this notation for string generation:
```
f"attrs.label = '{parsed.group(1)}'"
```
Note the 'f' before everything.